pkg/network/networker.go

```diff
@@ -501,6 +503,7 @@ func (n *networker) DisconnectPubTap(pubIPReservationID string) error {
 		return errors.Wrap(err, "could not load tap device")
 	}
 
+	// TODO: this will not work with a macvtap,
```

Comment:
I assume this is fixed?

Reply:
No, actually it's not fixed, because you can't unset the master of a macvlan. If u try that with command line it does not fail but doesn't change anything.
I am not sure what we can do here instead to disconnect the device. I thought may be change the txqueue back to 0

Reply:
right, I think honestly this is by design since the macvtap is essentially looks like a macvlan in the upper part. This function should just cut the network connection between the 2 ends of the interface somehow so a vm no longer has network access after calling this